erpnext/education/api.py

```diff
@@ -198,22 +198,6 @@ def get_fee_components(fee_structure):
 		return fs
 
 
-@frappe.whitelist()
-def get_fee_schedule(program, student_category=None):
```

Comment:
Let's not remove this API since it might be used in implementations. It's also used in demo creation:

https://github.com/frappe/erpnext/blob/7767dc1ee3536c72ae5db7eafaa387e07a7e9ac2/erpnext/demo/user/education.py#L51